website/docs/cloud-docs/overview/activate-flex.mdx

```diff
@@ -14,7 +14,7 @@ Follow these steps to activate HashiCorp Flex for a Standard or Plus Edition HCP
 
 There are several prerequisites for activating HashiCorp Flex:
 
-1. You must email the HCP organization ID with the HashiCorp sales team. 
```

Comment:
Hi @rkoron007 this pre req can be substituted with. “Create an HCP org.” Instructions. We no longer need to know the org id they will be using.